contracts/test/ITeleporterTokenBridgeTests.t.sol

```diff
@@ -0,0 +1,183 @@
+// (c) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+// SPDX-License-Identifier: Ecosystem
+
+pragma solidity 0.8.18;
+
+import {Test} from "forge-std/Test.sol";
+import {TeleporterRegistry} from "@teleporter/upgrades/TeleporterRegistry.sol";
+import {
+    ITeleporterMessenger,
+    TeleporterMessageInput,
+    TeleporterFeeInfo
+} from "@teleporter/ITeleporterMessenger.sol";
+
+import {
+    ITeleporterTokenBridge, SendTokensInput
+} from "../src/interfaces/ITeleporterTokenBridge.sol";
+import {IERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/IERC20.sol";
+import {SafeERC20} from "@openzeppelin/contracts@4.8.1/token/ERC20/utils/SafeERC20.sol";
+
+abstract contract ITeleporterTokenBridgeTest is Test {
```

Comment:
I think it's confusing to use the "I" convention typically used for interfaces here. Can it just be called `TeleporterTokenBridgeTest` as an abstract contract? (same for `INativeTokenBridgeTest"